### ISSUE TITLE
local test fails due to missing manifest/PyBOL

### ISSUE THREAD

**Issue by orbeckst** [Becksteinlab/MDPOW#64]
At the moment I cannot test locally out of the box.

I did a 

```
pip install --upgrade .
```

but because this does not install [PyBOL](https://github.com/ianmkenney/PyBOL) (manifest) running the tests locally fails. (See below)
- [x] make PyBOL pip-installable
- [x] add PyBOL to the requirements for tests in `setup.py` (something like `tests_require = [ ... ]`)

```
(develop) yngvi:MDPOW oliver$ py.test mdpow/tests/
============================================================= test session starts =============================================================
platform darwin -- Python 2.7.12, pytest-2.9.1, py-1.4.31, pluggy-0.3.1
rootdir: /Volumes/Data/oliver/Biop/Projects/Methods/MDPOW, inifile:
collected 49 items / 2 errors

mdpow/tests/test_emin.py ..
mdpow/tests/test_fep.py ..
mdpow/tests/test_forcefields.py ................................
mdpow/tests/test_runinput.py .........
mdpow/tests/test_solvation.py ....

=================================================================== ERRORS ====================================================================
__________________________________________ ERROR collecting mdpow/tests/test_equilibration_script.py __________________________________________
mdpow/tests/test_equilibration_script.py:3: in <module>
    import manifest
E   ImportError: No module named manifest
_______________________________________________ ERROR collecting mdpow/tests/test_fep_script.py _______________________________________________
mdpow/tests/test_fep_script.py:3: in <module>
    import manifest
E   ImportError: No module named manifest
===================================================== 49 passed, 2 error in 4.55 seconds ======================================================
```

**Comment by orbeckst** [comment]
See also #63.

**Comment by ianmkenney** [comment]
I fixed it for travis, but this will still be a problem for local tests until the PyPi package is built; I'll work on that now.

**Comment by orbeckst** [comment]
Was this fixed with PR #67 ?

**Comment by orbeckst** [comment]
If I 
```
pip install pybol
```
then I can run the tests
```
py.test -v mdpow
```
and it fails with
```
DEBUG:gromacs.core:gmx mdrun -c em.pdb -deffnm em -stepout 10 -v
__________________________________________________________ TestFEPScript.test_basic_run __________________________________________________________

self = <test_fep_script.TestFEPScript object at 0x10548c410>

    def test_basic_run(self):
        with in_dir(self.tmpdir.name, create=False):
            try:
                self._run_fep('water', 'benzene/')
            except Exception as err:
>               raise AssertionError('FEP simulations failed with exception:\n{0}'.format(str(err)))
E               AssertionError: FEP simulations failed with exception:
E               [Errno 1] Gromacs tool failed
E               Command invocation: gmx grompp -c /private/var/folders/qs/k07z49fh8xl6xd008k8wxhxr0000gp/T/tmpb9sqMl/benzene/Equilibrium/water/MD_NPT/md.gro -f md.mdp -pp processed.top -o md.tpr -n /private/var/folders/qs/k07z49fh8xl6xd008k8wxhxr0000gp/T/tmpb9sqMl/benzene/Equilibrium/water/solvation/main.ndx -p /private/var/folders/qs/k07z49fh8xl6xd008k8wxhxr0000gp/T/tmpb9sqMl/benzene/Equilibrium/water/top/pp_system.top -maxwarn 1

mdpow/tests/test_fep_script.py:38: AssertionError
```

Adding `pybol` to `test_requires` in `setup.py` does not automatically download it but I think that's ok.

**Comment by orbeckst** [comment]
I fixed this in 321527f79fdb0553872097484da0007314b7d57a by using https://pypi.python.org/pypi/pytest-runner.

One can now do
```
python setup.py test
```
to run the tests with `py.test`.
